tests/core/test_datamodules.py

```diff
@@ -201,47 +199,54 @@ def test_train_loop_only(tmpdir):
 
     trainer = Trainer(
         default_root_dir=tmpdir,
-        max_epochs=3,
+        max_epochs=1,
         weights_summary=None,
     )
 
     # fit model
     result = trainer.fit(model, dm)
-    assert result == 1
-    assert trainer.logger_connector.callback_metrics['loss'] < 0.6
+    assert result
+    # assert trainer.callback_metrics['loss'] < 0.6
 
 
 def test_train_val_loop_only(tmpdir):
     reset_seed()
 
-    dm = TrialMNISTDataModule(tmpdir)
+    dm = BoringDataModule()
+    model = BoringModel()
 
-    model = EvalModelTemplate()
     model.validation_step = None
     model.validation_step_end = None
     model.validation_epoch_end = None
 
     trainer = Trainer(
         default_root_dir=tmpdir,
-        max_epochs=3,
+        max_epochs=1,
         weights_summary=None,
     )
 
     # fit model
     result = trainer.fit(model, dm)
-    assert result == 1
-    assert trainer.logger_connector.callback_metrics['loss'] < 0.6
+    assert result
+    # assert trainer.callback_metrics['train_loss'] < 0.6
```

Comment:
what should be done with such assertions when using `BoringModel`? Since the dataset used with `BoringModel` is completely random.

Reply:
maybe we need a BoringMNISTModel for that 🤣

Reply:
we shall replace result validation with one end-to-end test..